packaging/ubuntu-focal/debian/amazon-ecs-init.amazon-ecs-volume-plugin.service

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Are these identical?  Can we use symlinks instead of copying?

Reply:
They are mostly identical but the path in the execstart command is slightly different than in the AL version